.vscode/settings.json

```diff
@@ -1,28 +1,5 @@
 {
-    // "autoDocstring.docstringFormat": "numpy",
-    "autoDocstring.customTemplatePath": "resources/docstring_template.mustache",
-    "cSpell.language": "en",
-    "cSpell.words": [
-        "automatable",
-        "autouse",
-        "BBGGRR",
-        "deinitialize",
-        "desynchronize",
-        "docformat",
-        "Enveditor",
-        "falsiness",
-        "indexedness",
-        "jsonschema",
-        "MIDICC",
-        "mypy",
-        "pdoc",
-        "ppqn",
-        "Precount",
-        "pyscript",
-        "Rect",
-        "Sysex",
-        "timebase"
-    ],
```

Comment:
Please undo these modifications, since I intend to keep the spell checker enabled.